### PR TITLE
frontend: Add fees tooltip text

### PR DESCRIPTION
**Pull request by nekhaly**
Resolves:  
#254

### CODE REVIEW

File: frontend/src/components/RequestFormInputs.vue [hunk 1]
```diff
@@ -228,8 +224,18 @@ const executeFaucetRequest = async () => {
 };
 const { active: faucetRequestActive, run: runFaucetRequest } =
   createAsyncProcess(executeFaucetRequest);
+
+const gasFeesTooltipText = `The fee amount is composed of three parts:
+  - the gas reimbursement fee
+  - the liquidity provider fee
+  - the Beamer service fee
+The gas reimbursement fee and the liquidity provider fee are paid out to the liquidity provider servicing the request, while the Beamer service fee stays with the contract and supports the Beamer platform development.
```
Comment (istankovic) [inline frontend/src/components/RequestFormInputs.vue:232]:
```suggestion
The gas reimbursement fee and the liquidity provider fee are paid out to the liquidity provider servicing the request, while the Beamer service fee stays with the contract and supports the Beamer platform development.

```

File: frontend/src/components/RequestFormInputs.vue [hunk 2]
```diff
@@ -228,8 +224,18 @@ const executeFaucetRequest = async () => {
 };
 const { active: faucetRequestActive, run: runFaucetRequest } =
   createAsyncProcess(executeFaucetRequest);
+
+const gasFeesTooltipText = `The fee amount is composed of three parts:
+  - the gas reimbursement fee
+  - the liquidity provider fee
+  - the Beamer service fee
```
Comment (istankovic) [inline frontend/src/components/RequestFormInputs.vue:231]: Can we replace these with bullet points? I think it would look nicer?

Comment (nekhaly) [reply frontend/src/components/RequestFormInputs.vue:231]: yes we can

File: frontend/src/components/RequestFormInputs.vue (hunk 1 above)
Comment (nekhaly) [reply frontend/src/components/RequestFormInputs.vue:232]: What is the suggestion here exactly, couldn't figure it out?

Comment (istankovic) [reply frontend/src/components/RequestFormInputs.vue:232]: To separate the middle paragraph with an empty line, above and below, in order to improve readability. Just my personal preference.

**Review by istankovic — APPROVED** [review]
Thanks!

**Pull request merged**